### DISCUSSION TITLE
`scala-cli fmt` and other commands  defaults

### DISCUSSION THREAD

**Discussion by romanowski** [VirtusLab/scala-cli#237] [General]
Currently, `scala-cli` formats the current directory when no inputs are given. This stick to the scalafmt defaults but breaks pattern known from other commands (like `run` or `compile`) when inputs are always needed.

Should we add `.` by default to `fmt` command (as we are doing it now) or stick to the pattern known from other commands? Maybe we should

**Comment by bjornregnell** [comment]
I guess it is easy to forget the dot after any command. I'd suggest to assume a missing arg means `.` on any command, but consistency is more important IMO. So I think we need to choose either to infer `.` in all commands if no args given or always require explicit `.` in all commands (the latter is a bit "boilerplaty" and the dot might be crytpic to a beginner).
